src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2696,17 +2697,8 @@ private boolean isOpenToCaller(String name, Class<?> caller) {
         return true;
     }
 
-    /**
-     * Returns the {@code ProtectionDomain} of this class.
-     *
-     * @return the ProtectionDomain of this class
-     *
-     * @see java.security.ProtectionDomain
-     * @since 1.2
-     */
-    public ProtectionDomain getProtectionDomain() {
-        return protectionDomain();
-    }
+    @Stable
+    private transient final ProtectionDomain protectionDomain;
```

Comment:
Isn't `@Stable` superfluous with a final field?

Reply:
Yes, I thought I removed it but that was probably the other PR.